src/community/release/ext-flatgeobuf.xml

```diff
@@ -0,0 +1,16 @@
+<assembly>
+    <id>flatgeobuf-plugin</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>release/target/dependency</directory>
+            <outputDirectory></outputDirectory>
+            <includes>
+                <include>gs-flatgeobuf*.jar</include>		
```

Comment:
You have to list all jars that are needed fro the flatgeobuf output format to work, that are not found in the basic GeoServer installation. At a minimum, I'd expect that the flatgeobuf java library you maintain is a requirement, maybe more?

Reply:
I see, I've added the dependencies but unsure if it's correct to add things that geotools/geoserver already depend on (i.e jts and perhaps json)

Reply:
Probably not... if you want to be sure, you can do the following:
* cd src/community
* mvn clean install -DskipTests -PcommunityRelease -T4 (in my case the best number for T is 3/4 the number of cores available)
* mvn assembly:attached -nsu

You'll find the zip for your module in target/release. Try to stick it into a GeoServer nightly build and see if it starts and works as expected. https://build.geoserver.org/geoserver/master/

Rinse and repeat until it does :-D

Reply:
Will do :)

Reply:
Yep that was a good exercise, found and fixed issues and successfully tested locally.